### ISSUE TITLE
ArgumentOutOfRangeException when finding text before stack view is populated

### ISSUE THREAD

**Issue by congyiwu** [Microsoft/perfview#547]
Repro w/ PerfView 2.0.2.0:
1. Open a stack view, e.g. "CPU Stacks" or "ThreadTime (with StartStop Activities) Stacks"
2. Quickly type something in the "Find" box and hit Enter before the view loads.

I saw an error dialog with the following stack trace:
```
System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
   at System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   at System.Collections.Generic.List`1.get_Item(Int32 index)
   at PerfView.CallTreeView.Find(String pat)
   at PerfView.StackWindow.FindNext(String pat)
   at PerfView.StackWindow.Find(String pat)
   at PerfView.StackWindow.DoFindEnter(Object sender, RoutedEventArgs e)
   at Controls.HistoryComboBox.FireEnter(Object sender, RoutedEventArgs e)
   at Controls.HistoryComboBox.DoKeyDown(Object sender, KeyEventArgs e)
   at System.Windows.Input.KeyEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.UIElement.RaiseEvent(RoutedEventArgs args, Boolean trusted)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndKeyboardInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawKeyboardActions actions, Int32 scanCode, Boolean isExtendedKey, Boolean isSystemKey, Int32 virtualKey)
   at System.Windows.Interop.HwndKeyboardInputProvider.ProcessKeyAction(MSG& msg, Boolean& handled)
   at System.Windows.Interop.HwndSource.CriticalTranslateAccelerator(MSG& msg, ModifierKeys modifiers)
   at System.Windows.Interop.HwndSource.OnPreprocessMessage(Object param)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by vancem** [comment]
This is trivial to fix (a test before the index operation).   I have it locally and will check it in when convenient (as this is not a blocking bug).

Thanks for reporting this!

**Comment by vancem** [comment]
This was fixed with commit 37df2535.
